launchable/commands/record/tests.py

```diff
@@ -311,6 +311,18 @@ def is_no_build(self) -> bool:
         def is_no_build(self, is_no_build: bool):
             self._is_no_build = is_no_build
 
+        @property
+        def data_builder(self) -> CaseEvent.DataBuilder:
```

Comment:
The CaseEvent.data_builder is suitable since it constructs the data for CaseEvent. However, this data_builder seems somewhat simplistic for the RecordTests class. Because this class will handle some "data". Does it make sense?

Reply:
Ok, let me rename it.

Reply:
[f945728](https://github.com/launchableinc/cli/pull/820/commits/f94572809244f24abe4849422a13ca036f002172)